### PR TITLE
Improve the heading for translated posts

### PR DESCRIPTION
**Pull request by j-f1**

### CODE REVIEW

**Comment by gaearon** [comment]
Screen before/after?

**Comment by j-f1** [comment]
| What | Before | After |
| -: | - | - |
| English | screen shot 2019-01-19 at 15 48 50 | screen shot 2019-01-19 at 15 48 20 |
| Italian | screen shot 2019-01-19 at 15 48 43 | screen shot 2019-01-19 at 15 48 31 |

**Comment by j-f1** [comment]
Feel free to suggest any copy changes.

**Comment by Saturn** [comment]
Would a dropdown select element be appropriate? To pick between translations. That way it might be easier to keep the top section clean and consistent.

File: gatsby-node.js [hunk 1]
```diff
@@ -50,6 +48,20 @@ exports.createPages = ({ graphql, actions }) => {
...
+          const plainSlug = slug.replace(/^\/[\w-]+\//, '/');
+          if (_originalTitleCache.has(plainSlug)) {
+            return _originalTitleCache.get(plainSlug);
+          }
+          const post = posts.find(({ node }) => node.fields.slug === plainSlug);
+          if (!post)
+            throw new Error('Can’t find parent for post with slug ' + slug);
+          const { title } = post.node.frontmatter;
+          _originalTitleCache.set(plainSlug, title);
+          return title;
```
Comment (gaearon) [inline gatsby-node.js]: Something like this could actually be useful for https://github.com/gaearon/overreacted.io/issues/99

**Pull request closed without merging**